XcodeServerSDKTests/TestUtils.swift

```diff
@@ -56,4 +58,87 @@ extension XCTestCase {
         let configuration = bot.configuration
         return configuration
     }
+}
+
+// MARK: Exception assertions
+// Based on: https://forums.developer.apple.com/thread/5824
+extension XCTestCase {
+    /**
+    Replacement method for XCTAssertThrowsError which isn't currently supported.
+    
+    - parameter message: Message which should be displayed
+    - parameter file:    File in which assertion happened
+    - parameter line:    Line in which assertion happened
+    - parameter block:   Block of code against which assertion should be matched
+    */
+    func XCTempAssertThrowsError(message: String = "", file: String = __FILE__, line: UInt = __LINE__, _ block: () throws -> ()) {
+        do {
+            try block()
+            
+            let msg = (message == "") ? "Tested block did not throw error as expected." : message
+            XCTFail(msg, file: file, line: line)
+        } catch {}
+    }
+    
+    /**
+    Replacement method for XCTAssertThrowsSpecificError which isn't currently supported.
+    
+    - parameter kind:    ErrorType which is expected to be thrown from block
+    - parameter message: Message which should be displayed
+    - parameter file:    File in which assertion happened
+    - parameter line:    Line in which assertion happened
+    - parameter block:   Block of code against which assertion should be matched
+    */
+    func XCTempAssertThrowsSpecificError(kind: ErrorType, _ message: String = "", file: String = __FILE__, line: UInt = __LINE__, _ block: () throws -> ()) {
+        do {
+            try block()
+            
+            let msg = (message == "") ? "Tested block did not throw expected \(kind) error." : message
+            XCTFail(msg, file: file, line: line)
+        } catch let error as NSError {
+            let expected = kind as NSError
+            if ((error.domain != expected.domain) || (error.code != expected.code)) {
```

Comment:
> Any ErrorType can be bridged to NSError with "as NSError".
-- <cite>[@jckarter](https://twitter.com/jckarter)</cite>

I think if  guys are suggesting this solution, we have to follow this right now... @esttorhe I hope they'll provide some built-in `XCTAssertThrow`-like methods before they release final version of Xcode 7.